packages/fluentui/react-builder/src/config.tsx

```diff
@@ -15,7 +15,78 @@ const projectPackageJson = require('@fluentui/react-northstar/package.json');
 const sandboxPackageJson = require('@fluentui/code-sandbox/package.json');
 const docsComponentsPackageJson = require('@fluentui/docs-components/package.json');
 
-export const EXCLUDED_COMPONENTS = ['Animation', 'Debug', 'Design', 'FocusZone', 'Portal', 'Provider', 'Ref'];
+export const EXCLUDED_COMPONENTS = [
+  'Animation',
+  'Debug',
+  'Design',
+  'FocusZone',
+  'Portal',
+  'Provider',
+  'Ref',
+  'DialogFooter',
+  'PopupContent',
+  'AlertDismissAction',
+  'AttachmentAction',
+  'AttachmentBody',
+  'AttachmentHeader',
+  'AttachmentDescription',
+  'AttachmentIcon',
+  'ChatMessageDetails',
+  'ChatMessageHeader',
+  'ChatMessageReadStatus',
+  'DatepickerCalendarCell',
+  'DatepickerCalendarHeader',
+  'DividerContent',
+  'HeaderDescription',
+  'SvgIcon',
+  'TooltipContent',
+  'FlexItem',
+  'ItemLayout',
+  'DropdownSelectedItem',
+  'DropdownSearchInput',
+  'FormFieldCustom',
+  'FormMessage',
+  'InputLabel',
+  'ButtonContent',
+  'MenuDivider',
+  'MenuItemIcon',
+  'MenuItemIndicator',
+  'MenuItemWrapper',
+  'MenuItemContent',
+  'SplitButtonToggle',
+  'ToolbarDivider',
+  'ToolbarItemIcon',
+  'ToolbarItemWrapper',
+  'ToolbarMenuDivider',
+  'ToolbarMenuItem',
+  'ToolbarMenuItemActiveIndicator',
+  'ToolbarMenuItemIcon',
+  'ToolbarMenuItemSubmenuIndicator',
+  'ToolbarMenuRadioGroup',
+  'ToolbarMenuRadioGroupWrapper',
+  'ToolbarRadioGroup',
+  'ToolbarItem',
+  'ToolbarCustomItem',
+  'ToolbarMenu',
+  'AccordionHeader',
+  'AccordionTitle',
+  'CardBody',
+  'CardHeader',
+  'CardFooter',
+  'CardColumn',
+  'CardExpandableBox',
+  'CardPreview',
+  'CardTopControls',
```

Comment:
How did you get the list? At least `Card` supports children API and these used to work